### PR TITLE
Added FB login

### LINKED ISSUE #65: Create Android FB login
We should enable users to login to the Android app via Facebook.

### PR DESCRIPTION
**Pull request by pisoiasul**
Fixes #65

### CODE REVIEW

File: WakeAndShake-android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -13,4 +13,5 @@
     <string name="permission_rationale">"Contacts permissions are needed for providing email
         completions."
     </string>
+    <string name="facebook_app_id">1135451059896869</string>
```
Comment (akellehe) [inline WakeAndShake-android/app/src/main/res/values/strings.xml]: These should be build specific... do we need to create dev/stage/prod?

Comment (pisoiasul) [inline WakeAndShake-android/app/src/main/res/values/strings.xml]: What do you mean ?

Currently it's the Dev fb app id, when we launch we'll replace with the prod fb app id

Comment (akellehe) [inline WakeAndShake-android/app/src/main/res/values/strings.xml]: Ideally we'd be able to test the app in all three environments by handling configuration variables. We have a dev fb app, and a stage one too at the moment.

Comment (pisoiasul) [inline WakeAndShake-android/app/src/main/res/values/strings.xml]: Well, exactly, this is a configuration variable.

**Pull request merged**